interview-questions.md

```diff
@@ -7,62 +7,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-  Your answer:
+  Your answer: test
 
-  Researched answer:
+  Researched answer: test
 
 
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: Git is a version control software. It lets multiple developers work on projects without putting said projects in jeopardy. Github is just a way to manage your project's "code history" in the cloud.
 
+  Researched answer: Git is a software use to track changes (think snapshots) to assist teams of developers during software development. Github is a cloud-based way to manage your repositories.
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: The JS operators that will return a Boolean value are <, >, =, <=, and >=.
 
-  Researched answer:
+  Researched answer: The following operators will all return a Boolean Value: <, >, ==, ===, <=, and >=.
```

Comment:
True, There are three operators that give Boolean values: logical, equality, relational